maptivist/src/main/webapp/index.html

```diff
@@ -89,6 +89,10 @@ <h1>Fill in Marker</h1>
                     <option value = "Other">Other</option>
                 </select>
                 <br>
+                <label for="marker-date"><b>Marker Date</b></label>
+                <br>
+                <input type="text" placeholder="dd/mm/yyyy" id="marker-date" name="marker-date" required>
```

Comment:
This is another case where I would probably use a name for the form field that was more precise:
e.g.

"marker-date-dd/mm/yyyy" for example. Although I don't think "/"s are allowed. So perhaps change it to "marker-date-dd-mm-yyyy" and change the placerholder to match.

Another thing to do would be to have a separate forms for month, day, and year. That way a user wouldn't accidentally type in mm/dd/yyyy instead.